clang/lib/APINotes/APINotesFormat.h

```diff
@@ -72,6 +72,8 @@ enum BlockID {
   /// information about the global function.
   GLOBAL_FUNCTION_BLOCK_ID,
 
+  MEMBER_FUNCTION_BLOCK_ID,
```

Comment:
MEMBER_FUNCTION_BLOCK_ID should go at the end as you can't renumber existing block IDs. Please also add a comment.